integration/rpc_test.go

```diff
@@ -61,7 +62,7 @@ func TestEventLogRPC(t *testing.T) {
 		if err != nil {
 			t.Logf("unable to establish skaffold grpc connection: retrying...")
 			attempts++
-			if attempts == retries {
+			if attempts == connectionRetries {
```

Comment:
why do we need to re-try here?  Is it because, the `--rpc-port` which the we ended up picking is not free any because some other test or CI job grabbed it?

Reply:
this would happen if skaffold hadn't started up the RPC server yet. it's pretty unlikely that we would ever actually need to retry here, I had this in the original client I wrote so I figured I would throw it in here for robustness.

Reply:
The number of connection re-tries is 20 which scares me. If there is a real issue, it would be masked. I am ok with a smaller number of re-tries here like 2. 
Another idea: we should log and collect metrics when retries happen. Can help us determine which skaffold code should we stabilize.

Reply:
I created one here https://github.com/GoogleContainerTools/skaffold/issues/1880

Reply:
HAHA yeah that does not need to be 20....I think I meant to set it to 2 originally. I'll change it here, and thanks for opening that issue